modules/gdpr_dump/src/Plugin/Gdpr/Sanitizer/RandomTextSanitizer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\gdpr_dump\Plugin\Gdpr\Sanitizer;
+
+use Drupal\Component\Utility\Random;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\gdpr_dump\Sanitizer\GdprSanitizerBase;
+
+/**
+ * Class TextSanitizer.
+ *
+ * @GdprSanitizer(
+ *   id = "gdpr_random_text_sanitizer",
+ *   label = @Translation("Random Text sanitizer"),
+ *   description=@Translation("Provides sanitation functionality intended to be
+ *   used for text fields.")
+ * )
+ *
+ * @package Drupal\gdpr_dump\Plugin\Gdpr\Sanitizer
+ */
+class RandomTextSanitizer extends GdprSanitizerBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function sanitize($input, FieldItemListInterface $field) {
+    if ($field != NULL) {
+      $max_length = $field->getDataDefinition()->getSetting("max_length");
+    }
+    else {
+      $max_length = NULL;
+    }
+
```

Comment:
You dont need to explicitly set `$max_length` to NULL as you are checking for `isset` later anyway. Just remove the else.

Reply:
Thanks - that's my C# background coming through!

Reply:
Done.